### PR TITLE
Spanish version

### PR DESCRIPTION
**Pull request by reuptaken**

### CODE REVIEW

**Comment by ghettodev** [comment]
Can you please remove the readme.md from this PR? It is the legacy version.

**Comment by reuptaken** [comment]
Removed READMEs

File: SUMMARY.md [hunk 1]
```diff
@@ -8,7 +8,9 @@
 * [FAQs](faqs/README.md)
   * [Español](faqs/es/README.md)
     * [Dai](faqs/es/dai.md)
+    * [Collateralized Debt Positions (CDP)](faqs/es/cdp.md)
     * [Governance](faqs/es/governance.md)
+    * [Stability Fee](faqs/es/stability-fee.md)
```
Comment (ghettodev) [inline SUMMARY.md]: Should these be in Spanish?

**Comment by reuptaken** [comment]
I took it from README, but I think it should be translated. I'll check with translator.

**Pull request closed without merging**